rust_icu_uloc/src/lib.rs

```diff
@@ -1244,4 +1262,9 @@ mod tests {
             "azerbaïdjanais (cyrillique, Azerbaïdjan, calendrier=calendrier hébraïque, t=it, usage privé=whatever)"
         );
     }
+    #[test]
+    fn test_get_available_locales() {
```

Comment:
Is there any fairly stable test for the contents of the returned vectors that you could add?

Reply:
I should be able to test for the existence of a few well-known locales. Will investigate.